<ui version="4.0" >
 <class>TraceDisplay</class>
 <widget class="QWidget" name="TraceDisplay" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget" >
      <attribute name="title" >
       <string>Samples</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="IvkPlot" native="1" name="valuePlot" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IvkTableWidget" name="valueTable" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize" >
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <row>
          <property name="text" >
           <string>Sample #</string>
          </property>
         </row>
         <row>
          <property name="text" >
           <string>Size</string>
          </property>
         </row>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Histogram</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="IvkPlot" native="1" name="histogramPlot" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Group Count:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSlider" name="groupSlider" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QSpinBox" name="groupSpin" >
         <property name="minimum" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Autocorrelation</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="IvkPlot" native="1" name="autocorrelationPlot" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Up to:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSlider" name="upToSlider" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QSpinBox" name="upToSpin" >
         <property name="minimum" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Significant values</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Sample count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="sampleCountEdit" >
        <property name="text" >
         <string>-</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Variance:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6" >
       <widget class="QLineEdit" name="varianceEdit" >
        <property name="text" >
         <string>-</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Average Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QLineEdit" name="averageSizeEdit" >
        <property name="styleSheet" >
         <string>background-color: #ccccee;</string>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Standard Deviation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="6" >
       <widget class="QLineEdit" name="standardDeviationEdit" >
        <property name="styleSheet" >
         <string>background-color: #cceecc</string>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IvkTableWidget</class>
   <extends>QTableWidget</extends>
   <header>widget/IvkTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>IvkPlot</class>
   <extends>QWidget</extends>
   <header>widget/IvkPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>upToSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>upToSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>759</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel" >
     <x>86</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upToSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>upToSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>192</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel" >
     <x>772</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>groupSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>656</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel" >
     <x>757</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>groupSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>756</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel" >
     <x>535</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
